### ISSUE TITLE
Crops grow like hell

### ISSUE THREAD

**Issue by DcNdrew** [InfinityRaider/AgriCraft#550]
Hey!
I use Agricraft, Hunger Overhaul, Harder Wildlife and Plant Mega Pack together.
Vanilla crops are okay, but Agricraft crops and PMP plants grow to mature in a day.
Do you know this issue?

Which log do you need?

**Comment by PitchBright** [comment]
This issue isn't Hunger Overhaul related
https://github.com/progwml6/HungerOverhaul/issues/118

**Comment by InfinityRaider** [comment]
Don't need a log, growth rates can be tweaked in the config.

**Comment by DcNdrew** [comment]
Before HW and/or HO worked on Agricraft crops.

**Comment by InfinityRaider** [comment]
what do you mean by that?

**Comment by PitchBright** [comment]
@InfinityRaider I don't think he's filing an actual bug report. I think he's mistaking Github for some sort of online tech support where people help him learn how to configure his mods. 

Plus he can't be bothered to spend the time to write something that uses a modicum of proper sentence structure so that the people that are trying to help him, aren't left wondering "wut?". That's why I referenced his original issue over on HO… I was hoping it would give you some context as to situation, and also some insight into the types of responses you'll get if you start trying to help this lad.

I'd say you can close this issue.

**Comment by InfinityRaider** [comment]
Well then, @DcNdrew, you get one more shot to properly explain your issue to me before I close this.

**Comment by DcNdrew** [comment]
Here we go: once upon a time I remember Harder Wildlife and mostly Hunger Overhaul forced Agricraft crops to grow slower, let them to have less fruits or something. As I remember, I even saw a comment about this feature. Well, now it doesn't work. I don't know which one should make the difference by default, that's why I wrote it everywhere.

**Comment by PitchBright** [comment]
Hunger Overhaul does not affect crop growth rate.

"Once upon a time" really doesn't give us a frame of reference to work with. Do you mean "back in the day minecraft 1.2.5 once upon a time"?

or

"Last week before I borked my minecraft folder once upon a time"?

I think the real problem where is that you're not describing exactly what you are expecting to happen. Are you saying that Crops are growing at a vanilla rate, and you were expecting them to grow slower because of some combination of mods you've installed.

I also think you're not understanding what Github Issue tracker is for. This is a place where you come to report issues. Typically issues that you're either able to show some form of evidence of, either via screencaptures, crash reports or… helluvagooddescription.

You're basically posting your problem in the githubs of the mods you're using and just saying "I'm using your mods and I'm having a problem."

What you want is some "tech support". Go to the IRC channel of the mod that you know the issue to be the cause of your problems, and post your question or problem on there, and wait for somebody to reply… for that sorta thing.

**Comment by InfinityRaider** [comment]
I'm not sure if I changed anything about that, but I don't believe so. The AppleCore growth tick event is still being fired: https://github.com/InfinityRaider/AgriCraft/blob/1.7.10/src/main/java/com/InfinityRaider/AgriCraft/blocks/BlockCrop.java#L100-L101

And this is both used by Reasonable Realism and Hunger Overhaul, so I don't see why it shouldn't work.
Also there's a global growth rate modifier in the agricraft config, so you could also use that.

**Comment by DcNdrew** [comment]
InfinityRaider: thank you, so you say it should work, but it doesn't. That's why I asked about logs. If there's any clue in them I can search for that too.

I thought I can set growth rate in Agricraft config, but if the global growth rate doesn't work, that's a real issue. 

PitchBright: you could save your time if you tried to understand the problem possible compatibility, instead of lecturing me.

**Comment by InfinityRaider** [comment]
As I said before, logs won't help here, and can you specify what you mean with "the global growth rate doesn't work"

**Comment by PitchBright** [comment]
@DcNdrew 

My bad. 3 Million people using these mods together, and you're the first person to notice that they're incompatible. That's probably why "possible compatibility" problem didn't even occur to me.

**Comment by DcNdrew** [comment]
InfinityRaider:
As soon as I can play I'll tell you more. I'm in 12 hours in night shift for 3 days in a row, after 2 hours of travel I got home for 6 hours to sleep. 
Until then I can tell you what I experienced: vanilla  plants and for example flax grow real slow, just how they should. But when I put vanilla on Agricraft crops, they start to grow and became mature nearly in a day. 

PitchBright: no problem, I should be more specific, but I thought the coders may know how and what they use, which funcions are responsible for growing. So it sounds that may AppleCore can do with it.
